.github/workflows/test.yaml

```diff
@@ -4,10 +4,6 @@ on:
   pull_request:
     branches:
       - main
-    paths:
-      - "otaclient/**"
-      - "tests/**"
-      - ".github/workflows/test.yaml"
```

Comment:
github currently doesn't support a clean way to manually trigger test CI on a PR. test CI is required for merging, but path filters here will skip the test CI if no changes occur on paths, blocking non-code PR to be merged.